Homework5/Task3Vacation/VacationData.cs

```diff
@@ -0,0 +1,108 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Task3Vacation
+{
+    class VacationData
+    {
+        public List<(string name, DateTime start, DateTime finish)> VacationDates { get; private set; }
+
+        public VacationData()
+        {
+            VacationDates = new();
+        }
+
+        public void AddVacation((string, DateTime, DateTime) vacation)
+        {
+            VacationDates.Add(vacation);
+        }
+
+        public double AverageVacationDuration()
+        {
+            return VacationDates.Average(x => (x.finish - x.start).Days);
+        }
+
+        public List<(string, double)> AverageEmployeeVacationDuration()
+        {
+            List<(string, double)> averageByEmployee = VacationDates.GroupBy(e => e.name)
+                                                .Select(g => ((Name: g.Key, Avg: g.Average(x => (x.finish - x.start).Days)))).ToList();
+            return averageByEmployee;
+        }
+
+        public void EmployeesOnVacationByMonth()
+        {
+            var avgByEmpl = VacationDates.Where(x => (x.finish - x.start).Days > 1)
+                                .GroupBy(vac => vac.start.Month)
```

Comment:
Given a person with 2 vacations in the same month. 
Current code returns 2 records per month for him, however it should return 1, because per requirements method should return number of employees per month, not number of a vacations.